server/project/project.go

```diff
@@ -37,7 +37,7 @@ func (s *Server) Create(ctx context.Context, q *ProjectCreateRequest) (*v1alpha1
 		return nil, grpc.ErrPermissionDenied
 	}
 	if q.Project.Name == common.DefaultAppProjectName {
-		return nil, status.Errorf(codes.InvalidArgument, "name '%s' is reserved and cannot be used as a project name")
+		return nil, status.Errorf(codes.InvalidArgument, "name '%s' is reserved and cannot be used as a project name", common.DefaultAppProjectName)
```

Comment:
Do we have a check for this on API server? If so, I think we can defer this to API server and skip the client side check. This way our error will be consistent.

Reply:
We cannot have exact same checks in API, since API does not know if users trying to add or remove the destination. But it make sense check for duplicates on API side (implemented). I think it make sense to keep checks in CLI since we can report better error messages